MbuguaCaleb/challangetwo-testing--repo
# [starts#163080248] update folder structure

Sets up the folder /package structure for versioning my apis and running packages
adds and init.py file to each module

How should this be manually tested?

Clone this repository on your machine:
git clone https://github.com/MbuguaCaleb/gitchallangetwo-testing--repo

CD into the project folder on your machine
$ cd gitchallangetwo-testing--repo


Check out the branch for this PR
$ git checkout ch-folder-structure-163080248

**Relevant pivotal tracker story associated with this PR
#163080248

**screenshots**
![screenshot from 2019-01-09 10-08-00](https://user-images.githubusercontent.com/45809091/50882674-9984a380-13f6-11e9-9de9-ea632c6ed647.png)
